docs/developers/git-workflow.rst

```diff
@@ -754,14 +754,29 @@ Invenio git commit log messages are usually formatted in the following
 way:
 
 * commit message headline providing short summary (maximum 50 chars)
-  formatted in the style of ``ModuleName: short description``. (using mostly
-  nouns, no verbs);
+  formatted in the style of ``component: short description``. (using
+  mostly nouns, no verbs);
 
 * empty line;
 
-* commit message body with a detailed description of what this patch
-  does, formatted as a bulletted list, with one empty line between
-  items (using present tense).
+* commit message body with a detailed description of what this patch does,
+  formatted as a bulletted list, with one empty line between items (using
+  present tense), and leading label for bullets with *important* messages
+  that should be included in release notes.
```

Comment:
From here it's not clear what to do with non-important messages. Can you explicitly add that non-important messages can still be added but without prefixes?